### PR TITLE
Update polyserve dep to 0.16.0

### PR DESCRIPTION
**Pull request by justinfagnani**
- [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
     "mocha": "^3.1.2",
     "multer": "^1.1.0",
     "nomnom": "^1.8.1",
-    "polyserve": "0.16.0-prerelease.9",
+    "polyserve": "0.16.0",
```
Comment (rictic) [inline package.json]: ^0.16.0 for semver?

Comment (justinfagnani) [inline package.json]: oh yeah...

**Comment by stramel** [comment]
Can we close this since the `package.json` is now updated to `^0.19.0`?

**Comment by justinfagnani** [comment]
Yep, thanks for noticing!

**Pull request closed without merging**